### DISCUSSION TITLE
Next.js run time logs not showing

### DISCUSSION THREAD

**Discussion by kennyegun24** [vercel/community#6812] [Account Issue] [answered]
### Question

I recently deployed two Next.js projects on my vercel account, and I noticed that the run time logs are always empty. I added some console logs statements in my code to check out some issues with my database in production environment, but I can't even see the logs
![image](https://github.com/vercel/community/assets/109461921/d3b8cf1b-35dd-4944-aa04-5b6bca9366f7)

**Comment by versecafe** [comment]
@kennyegun24 Can you share your project & the repository?

**Reply by kennyegun24** [reply]
@versecafe , thanks for responding


Here is my repository
https://github.com/kennyegun24/reiki-crm

There happens to be an issue with data caching at the backend also.... In my local development server, each request queries the database for new data... in production mode, it just keeps sending cached data... here are images to show you what I mean

![image](https://github.com/vercel/community/assets/109461921/a1617213-c0e1-4db0-a248-a3e608007f67)
![image](https://github.com/vercel/community/assets/109461921/782c9836-7ae9-44b6-9a40-3b3049df6f58)

In the first screenshot, it happens to be fetched from my local server... I just deleted one object from my mongo db database, hence I only have 3 objects in therfe...
But in the second screenshot, which is taken from the prod server, it still shows that there are 4 objects, instead of just 3

Here is a snippet of my code to fetch from database
![image](https://github.com/vercel/community/assets/109461921/3aab4ccb-ef5f-4590-9bce-7ac59d3833c4)

**Comment by versecafe** [comment, marked as answer]
@kennyegun24 you're making a GET request with no param changes so it should be cached by default in production, you can try something like

```ts
export const dynamic = 'force-dynamic';
```

in the routes you want to always rerun or instead use cache headers like this to have set revalidation time
```json
{
  "Cache-Control": "max-age=300, s-maxage=300, stale-while-revalidate",
}
```

**Reply by kennyegun24** [reply]
@versecafe thank you, the caching has stopped
I can also see the runtime logs now

Thank you for resolving my issues 🙇
